### DISCUSSION TITLE
andriod compile problems

### DISCUSSION THREAD

**Discussion by Jwy-jump** [rustdesk/rustdesk#5747] [General]
`error: failed to run custom build command for scrap v0.5.0 (/rustdesk/rustdesk/libs/scrap)

Caused by:
  process didn't exit successfully: `/rustdesk/rustdesk/target/release/build/scrap-6b55e008504b03c4/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=NO_PKG_CONFIG_libyuv
  cargo:info=arm64-android
  cargo:rustc-link-lib=static=yuv
  cargo:rustc-link-search=/rustdesk/vcpkg/installed/arm64-android/lib
  cargo:include=/rustdesk/vcpkg/installed/arm64-android/include
  cargo:rerun-if-env-changed=NO_PKG_CONFIG_libvpx
  cargo:info=arm64-android
  cargo:rustc-link-lib=static=vpx
  cargo:rustc-link-search=/rustdesk/vcpkg/installed/arm64-android/lib
  cargo:include=/rustdesk/vcpkg/installed/arm64-android/include
  rerun-if-changed=/rustdesk/rustdesk/libs/scrap/src/bindings/vpx_ffi.h
  rerun-if-changed=/rustdesk/vcpkg/installed/arm64-android/include
  cargo:rerun-if-env-changed=NO_PKG_CONFIG_aom
  cargo:info=arm64-android
  cargo:rustc-link-lib=static=aom
  cargo:rustc-link-search=/rustdesk/vcpkg/installed/arm64-android/lib
  cargo:include=/rustdesk/vcpkg/installed/arm64-android/include
  rerun-if-changed=/rustdesk/rustdesk/libs/scrap/src/bindings/aom_ffi.h
  rerun-if-changed=/rustdesk/vcpkg/installed/arm64-android/include

  --- stderr
  /rustdesk/rustdesk/libs/scrap/src/bindings/aom_ffi.h:1:10: fatal error: 'aom/aom.h' file not found
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ClangDiagnostic("/rustdesk/rustdesk/libs/scrap/src/bindings/aom_ffi.h:1:10: fatal error: 'aom/aom.h' file not found\n")', libs/scrap/build.rs:163:18
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
note: If the build failed due to a missing target, you can run this command:
note: 
note:     rustup target install aarch64-linux-android`

**Comment by parazit-IRAN** [comment]
How did you fix it?
